### ISSUE TITLE
lofty::mp3::read::read_from panics on broken mp3 file

### ISSUE THREAD

**Issue by swick** [Serial-ATA/lofty-rs#39]
amberol calls `lofty::probe::read_from_path` which results in a panic.
```
#6  0x00005555555a79bd in core::panicking::panic () at library/core/src/panicking.rs:48
#7  0x0000555555605b6b in lofty::mp3::read::read_from ()
#8  0x00005555555dcb5a in lofty::probe::read_from_path ()
#9  0x00005555555beb9e in amberol::audio::song::SongData::from_uri ()
```

The file ([11 22:22.zip](https://github.com/Serial-ATA/lofty-rs/files/8549989/11.22.22.zip)) is broken but it still should not panic.

**Comment by Serial-ATA** [comment]
Thanks for the report! That file is just an ID3v2 tag with no MPEG frames. Looks like that's a case I somehow missed. I'll get that fixed.

**Comment by Serial-ATA** [comment]
Fixed in https://github.com/Serial-ATA/lofty-rs/commit/89ebe2ed17c5a52ba95f58fdc0334f3511379e47. Published in 0.6.2, please inform the maintainer of Amberol :)
